level.go

```diff
@@ -75,8 +75,8 @@ var ErrNotFound = errors.New("entcache: entry was not found")
 type (
 	// LRU provides an LRU cache that implements the AddGetter interface.
 	LRU struct {
+		mu *sync.RWMutex
```

Comment:
```suggestion
		mu sync.RWMutex
```

The pointer here is not really needed, as all methods get pointer receiver.